languages/go/exercises/concept/booleans/.docs/introduction.md

```diff
@@ -10,7 +10,7 @@ var a bool
 Go supports three logical operators that can evaluate expressions down to Boolean values, returning either `true` or `false`.
 
 | Operator   | What it means                              |
-| ---------- | ------------------------------------------ | ------ | --------------------------------------------- |
+| ---------- | ------------------------------------------ |
 | `&&` (and) | It is true if both statements are true.    |
 | `          |                                            | ` (or) | It is true if at least one statement is true. |
```

Comment:
I think this is wrong.

Reply:
I think that was already wrong in their doc. It looked misformatted to me when they merged it.

Reply:
let's merge it then?